src/sider/senoppfolging/SenOppfolging.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { ReactElement } from "react";
+import { BodyShort, Box, Heading, Label, List } from "@navikt/ds-react";
+import { useSenOppfolgingSvarQuery } from "@/data/senoppfolging/useSenoppfolgingSvarQuery";
+import { tilLesbarDatoMedArUtenManedNavn } from "@/utils/datoUtils";
+
+const texts = {
+  heading: "Sykmeldtes svar",
+  oppfolging: "Vurder videre oppfølging",
+  vurderFolgende: "Basert på svarene kan du vurdere følgende",
+  rutine: [
+    "14a-vedtak: Hvis den sykmeldte har svart at han eller hun trenger oppfølging, skal det gjøres et 14a-vedtak i Arena",
+    "Dialogmøte: Vurder om det burde kalles inn til et dialogmøte",
+    "Kontakt bruker: Ta kontakt med den sykmeldte for å avklare behov for videre oppfølging",
+    "Kontakt behandler: Ta kontakt med behandler for å innhente legeerklæring",
+    "AAP: Vurder om den sykmeldte bør søke om AAP",
+  ],
+  ikkeVarslet: {
+    info1:
+      "Den sykmeldte har ikke mottatt varsel om Snart slutt på sykepengene enda.",
```

Comment:
Er det rart med stor bokstav midt i her? Kanskje det er best å skrive det i klartekst uten å omtale det som et konsept 😊
```suggestion
      "Den sykmeldte har ikke mottatt varsel om at det snart er slutt på sykepengene enda.",
```